frontend/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contexture - Managing your Domains & Contexts</title>
+    <script src="assets/js/Contexture.js"></script>
+    <link rel="stylesheet" href="assets/css/contexture.css">
+</head>
+<body>
+<script>
+    Elm.Main.init();
+</script>
+<script src="node_modules/jquery/dist/jquery.slim.min.js"></script>
```

Comment:
I think we don't need `jQuery` in the HTML?

Reply:
Good question, I think we have to decide whether or not we want to re-implement stuff like the accordion (https://getbootstrap.com/docs/5.0/components/accordion/) that you have out of the box otherwise (and is going to be used in different parts of the application

Reply:
Hm yeah, I guess behavior is pretty limited and we don't use a lot of specialized components. For popovers I'd still want to take a dependency on Popper (or an equivalent Elm package), but let's look at that later. Will pull the dependencies out!